source/includes/api/_metadata.md

```diff
@@ -1,240 +1,537 @@
+When it comes to its internal structure, metadata objects have a balance of structured and free-format fields that promote a common base across all objects, while allowing different applications and use cases to have their own specific structure and details. You can learn more about the available fields in the [metadata reference](#metadata-reference) section.
 
-<aside class="notice">
-Please, be sure that the request is properly authenticated and the current user has permission to the resource.
-If you don't know how to do this, please go to the <a href="#authentication">Authentication section</a>
-</aside>
+Last but not least, it's important to keep in mind that the behavior of metadata objects and endpoints aims to be, as much as possible, independent from the target resource it references. In the detailed endpoint documentation below we'll cover the different endpoints in depth, and highlight the differences in behavior when handling different resource types, but you can safely assume that, for most of it, behavior described for a type of resource will be the same for all 3 types.
 
-> Create metadata for a dataset
+## Metadata endpoints overview
+
+As covered above, each metadata object directly concerns a single dataset, widget or layer. The endpoint structure mostly reflects that, with each metadata action (getting, creating, modifying or deleting metadata) being available through 3 different endpoints, one for each type of resource. These endpoints will be documented as a single element, as they behave in exactly the same way, with the only difference between them being the actual endpoint URL. Any differences that may exist will be pointed out, but those will be rare. The only exceptions are the endpoints to load all metadata (which ignores resource type) and to clone a dataset's metadata, which doesn't have a widget or layer equivalent.
+
+Any user can retrieve metadata objects, however to create, update or delete and object the appropriate [authentication](#authentication) credentials are required:
+
+- You must be a user with role:
+    - `ADMIN`, or
+    - `MANAGER` and own the metadata resource you are trying to manage.
+- You must belong to the same `application` as the resource associated with the metadata object.
+
+> Overview of error messages
+
+| Error code   | Error message          | Description |
+-------------- | ---------------------- | --------------
+| 400          | Bad request            | Query or body parameters are not valid. |
+| 400          | Bad request            | "Metadata of resource dataset: `<dataset-id>`, application: `<application>` and language: `<language>` already exists". |
+| 401          | Unauthorized           | The token provided is not recognized or is invalid. |
+| 403          | Authorization failed.  | Your user account does not have an `ADMIN` or `MANAGER` role or belong to the same `application` group as the resource associated with the metadata object. |
```

Comment:
This table feels a bit lost here, I could use either some introductory text or a better description of when these errors can occur.

Reply:
yeah, this table was compiled by Ed, and I wanted to remove it, but not without creating new ones per endpoint, which I did, but then forgot to delete this. Thanks